crates/bevy_render/src/camera/camera.rs

```diff
@@ -376,6 +376,8 @@ pub enum RenderTarget {
     Window(WindowRef),
     /// Image to which the camera's view is rendered.
     Image(Handle<Image>),
+    /// Texture View to which the camera's view is rendered.
+    TextureView(ManualTextureViewHandle),
```

Comment:
Might be worth adding a doc comment about the use case since it's been asked a few times now and can be useful context for why you would use one over the other.

```suggestion
    /// Texture View to which the camera's view is rendered.
    /// Useful when the texture view needs to be created outside of Bevy, for example OpenXR. 
    TextureView(ManualTextureViewHandle),
```